src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private final Random random = new Random();
+    private final ColorSupplier supplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Color randomColor = supplier.getRandomColor();
+        Ball ball = new Ball();
+        ball.setColor(randomColor);
+        ball.setNumber(random.nextInt(100));
```

Comment:
```suggestion
        ball.setNumber(random.nextInt(MAX_NUMBER));
```
lets add constant - dont use magic numbers